cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/FieldAttributes/RecordEnumAttributes.tsx

```diff
@@ -41,13 +42,13 @@ function RecordEnumTypeAttributes({
       <div className={classes.root}>
         <WidgetWrapper
           pluginProperty={{
-            name: 'doc',
+            name: 'documentation',
             macroSupported: false,
             description: 'documentation for the record',
```

Comment:
capitalize for consistency?

Reply:
This should be ok since none of the plugin properties are being used except the description. The API for the `WidgetWrapper` is tied to plugins closely. The name and macro are not being used since none of them are being used.